detections/network/cisco_secure_firewall___bits_network_activity.yml

```diff
@@ -0,0 +1,72 @@
+name: Cisco Secure Firewall - Bits Network Activity
+id: b08e69d4-b42d-494c-bd30-abaaa3571ba4
+version: 1
+date: '2025-04-01'
+author: Nasreddine Bencherchali, Splunk
+status: production
+type: Anomaly
+description: The following analytic detects the use of the Background Intelligent Transfer Service (BITS) client application in allowed outbound connections. It leverages logs from Cisco Secure Firewall Threat Defense devices and identifies instances where BITS is used to initiate downloads from non-standard or unexpected domains. While BITS is a legitimate Windows service used for downloading updates, it is also commonly abused by adversaries to stealthily retrieve payloads or tools. This analytic filters out known Microsoft Edge update URLs and focuses on connections that may indicate suspicious or unauthorized file transfers. If confirmed malicious, this could represent a command and control (C2) channel or a download of malware or tooling as part of an attack chain.
+data_source:
+- Cisco Secure Firewall Threat Defense Connection Event
+search: |
+  `cisco_secure_firewall` EventType=ConnectionEvent action=Allow ClientApplication="BITS" AND NOT url IN ("*://msedge.b.tlu.dl*")
+  | stats count min(_time) as firstTime max(_time) as lastTime by src_ip, dest, dest_port, transport, rule, url, EVE_Process, ClientApplication, ClientApplicationVersion 
```

Comment:
thinking out loud here :should we consider renaming EVE_Process to process_name : this may be better if we are thinking of correlating these with Endpoint detections

Reply:
Would need to discuss that field and others with the FTD/NTDR teams. The values are not always process like. For now lets leave it as is if thats okay.